src/plugins/validation/2and3/semantic-validators/pagination-ibm.js

```diff
@@ -0,0 +1,181 @@
+module.exports.validate = function({ jsSpec }, config) {
+  const result = {};
+  result.error = [];
+  result.warning = [];
+
+  config = config.pagination;
+  const queryParameters = [];
+  for (const head in jsSpec.paths) {
+    if (/{/.test(head)) {
+      continue;
+    }
+    if (jsSpec.paths[head].get) {
+      for (const resp in jsSpec.paths[head].get.responses) {
+        for (const con in jsSpec.paths[head].get.responses[resp].content) {
+          if (con == 'application/json') {
+            let arrayOnTop = false;
+            if (
+              (jsSpec.paths[head].get.responses[resp].content[con].schema &&
+                jsSpec.paths[head].get.responses[resp].content[con].schema
+                  .properties) ||
+              (jsSpec.paths[head].get.responses[resp].content[con].schema
+                .pagination &&
+                jsSpec.paths[head].get.responses[resp].content[con].schema
+                  .pagination.properties)
+            ) {
+              for (const prop in jsSpec.paths[head].get.responses[resp].content[
+                con
+              ].schema.properties) {
+                if (
+                  jsSpec.paths[head].get.responses[resp].content[con].schema
+                    .properties[prop].type == 'array'
```

Comment:
You should be using `===` rather than `==` pretty much everywhere.

A good article on this: https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a
> Triple Equals is superior to double equals. Whenever possible, you should use triple equals to test equality.